pkg/tindex/inmem_test.go

```diff
@@ -120,7 +150,8 @@ func TestGetJournals(t *testing.T) {
 	defer os.RemoveAll(dir) // clean up
 
 	ims := NewInmemService().(*inmemService)
-	ims.Config = &InMemConfig{WorkingDir: dir, CreateNew: true, DoNotSave: true}
+	ims.JCtrlr = &testJrnlCtrlr{}
```

Comment:
Nitpick: `JCtrlr` was pretty hard for me to read , may be just `ims.Journal`? The fact that it's a controller is communicated by the value's interface.

Reply:
same for the implementation, e.g. `&testJournal` reads way easier to me

Reply:
@klizhentas agree that it looks awkwardly. I would rather call it in plural like `ims.Journals` then